raw_datasets.json

```diff
@@ -179,5 +179,110 @@
+  ],
+  "louisina_state_university_hsc_monroe_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisina_state_university_hsc_no_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisina_state_university_hsc_shreveport_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisina_state_university_hsc_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisiana_state_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisina_state_university_shreveport_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "louisiana_technical_university_pd": [ 
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "loyola_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "mcneese_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "nicholls_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "northwestern_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "northwestern_state_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "nunez_community_college_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "southeastern_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "southern_university_baton_rouge_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "southern_university_new_orleans_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "southern_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "southern_university_shreveport_pd": [
+   "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1" 
+  ],
+  "tulane_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "university_of_louisiana_lafayette_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "university_of_louisiana_monroe_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "university_of_new_orleans_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "university_of_southwestern_louisiana_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
+  ],
+  "xavier_university_pd": [
+    "https://www.dropbox.com/s/f0m3ym4vvk66134/pprr_post_2020_11_06.csv?dl=1"
```

Comment:
Duplicating the same file across multiple folders seems wasteful to me. Duplicating the same script over many files just changing the agency also seems wasteful. We don't have to stick to the implicit convention of one script/source file per agency if it is going to cause performance problems later. I would argue that the convention should be one clean script per one source file or a couple of highly related source files. It's not enough if we just clean the data, we also have to make sure the code remains easy to maintain and runs fast.

Reply:
That makes sense. I guess that I was thinking ahead in that we'll eventually have distinct inputs for each university because I expect to receive roster data, eventually, and so I wanted to only use post as a place holder so that we can tag news articles to officers. But it makes a lot more sense to build out those entries as we receive data. I'll make the changes.